api/internal/core/entity/entity.go

```diff
@@ -220,3 +220,12 @@ type Script struct {
 	ID     string      `json:"id"`
 	Script interface{} `json:"script,omitempty"`
 }
+
+type ServerInfo struct {
+	BaseInfo
+	LastReportTime int    `json:"last_report_time,omitempty"`
```

Comment:
By convention we use `int64` to store timestamp.

Reply:
Fixed.